.openpublishing.redirection.standard.json

```diff
@@ -808,6 +808,10 @@
             "source_path_from_root": "/docs/standard/using-linq.md",
             "redirect_url": "/dotnet/standard/linq/",
             "redirect_document_id": true
+        },
+        {
+            "source_path_from_root": "/docs/standard/native-interop/cross-platform.md",
+            "redirect_url": "/dotnet/standard/native-interop/pinvoke"
```

Comment:
Shouldn't this point to native-library-loading?

Reply:
Fixed. I was not sure whether it is better to make it point to the renamed doc or the general doc.